addons/website_mass_mailing/static/src/snippets/s_popup/000.js

```diff
@@ -0,0 +1,27 @@
+/** @odoo-module **/
+
+import PopupWidget from 'website.s_popup';
+
+PopupWidget.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+	/**
+     * Prevents the (newsletter) popup to be shown if the user is subscribed.
+	 *
+     * @override
+     */
+    _canShowPopup() {
+        if (
+            this.$el.is('.o_newsletter_popup') &&
+            this.$el.find('input.js_subscribe_email').prop('disabled')
+        ) {
```

Comment:
:roll_eyes: If you want :smile:

Reply:
I had a hard time choosing, I think I will stick to this style :D